dag/proptest-regressions/node_dag.txt

```diff
@@ -0,0 +1 @@
+cc fa3519710f9f7dc108a638115d5d25e9086a6feacc75b01f5576b4a09061cc3b
```

Comment:
Is this needed? Is this a test output?

Reply:
This is an Rng seed. It will make sure that we do not regress on this particular case (e.g. removing the prop_assume) by incentivizing the Rng to execute at least the case that found a bug in #243.